dwi_ml/io/TO_SEND_TO_SCILPY.py

```diff
@@ -1,97 +0,0 @@
-"""I/O related utilities."""
-from __future__ import annotations
-import pathlib
-from typing import Tuple
-
-import nibabel as nib
-import numpy as np
-from dipy.core.gradients import GradientTable, gradient_table as load_gradient_table
-
-from scil_vital.shared.code.transformation.volume import resample_volume
-
-
-def load_dwi(dwi_file: str) -> nib.Nifti1Image:
```

Comment:
I couldn't find the functions in this file; I'm sure they exist somewhere in scilpy, what would their names be ?

Reply:
Thanks for pointing this out. We haven't really dealt with this function, it's true.

Hum... Actually, the only real line here is nib.load. The rest is a check of the name of the file. It shouldn't really be an "official" function, just as part of a script, I think.

I will bring back this file but rename it to "to_clean".